types/node/index.d.ts

```diff
@@ -40,6 +40,7 @@
 //                 NodeJS Contributors <https://github.com/NodeJS>
 //                 Linus Unnebäck <https://github.com/LinusU>
 //                 wafuwafu13 <https://github.com/wafuwafu13>
+//                 Alek Evans <https://github.com/AlekEagle>
```

Comment:
Just confirm, this is not for allocating credit - git does that - this is a list of people who can devote time to reviewing changes to these types.  Was that your intention?

Reply:
I thought it was just for credit, I'll be sure to remove it.